src/api-helper/log/LoggingProgressLog.ts

```diff
@@ -28,7 +28,7 @@ export class LoggingProgressLog implements ProgressLog {
     }
 
     public write(pWhat: string) {
-        let what = pWhat;
```

Comment:
I generally use a convention of the parameter having the natural name and the implementation using `<naturalName>ToUse`